plugins/modules/nd_snapshot.py

```diff
@@ -29,76 +31,81 @@
     type: str
     default: default
     aliases: [ fab_name, ig_name ]
-  site:
+  fabric:
     description:
-    - Names of the Assurance Entity.
+    - The name of the fabric.
     type: str
     required: true
+    aliases: [ fabric_name, site, site_name ]
   period:
     description:
-    - Epoch period.
+    - The snapshot/epoch period.
```

Comment:
Is mentioning snapshot/epoch everywhere too much? Shall we just mention this once in the main description like you already did?